configurations/sharepoint-woo/synchronizations/xxllnc-suite-to-publications.json

```diff
@@ -13,6 +14,7 @@
         "headers": [],
         "query.startdatum__gte": "2024-08-01",
         "query.einddatum__lt": "2025-01-01",
+        "query.maximaleVertrouwelijkheidaanduiding": "openbaar",
```

Comment:
Moet "aanduiding" niet met een hoofdletter?
ik ga ervan uit dat dit met een hoofdletter moet zijn want de andere worden in deze string hebben ook hoofdletters

Reply:
Nee dit is een veld vanuit xxllnc suite, ik heb het niet verzonnen